nipype/interfaces/base.py

```diff
@@ -1627,7 +1612,7 @@ def _overload_extension(self, value, name=None):
     def _list_outputs(self):
         metadata = dict(name_source=lambda t: t is not None)
         traits = self.inputs.traits(**metadata)
-        if traits:
+        if traits and getattr(self, 'output_spec', None) is not None:
             outputs = self.output_spec().get()
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [self.output_spec is not callable](https://www.codacy.com/app/krzysztof-gorgolewski/nipy_nipype/file/1967592315/issues/source?bid=2767207&fileBranchId=2767718#l1616)